button/button.css

```diff
@@ -0,0 +1,116 @@
+button,
+a.button {
+  background-color: var(--grey-90-a10);
+  border: 0 none;
+  border-radius: 2px;
+  color: var(--grey-90);
+  display: inline-block;
+  font-family: inherit;
+  font-size: 13px;
+  font-weight: 400;
+  height: 32px;
+  line-height: 32px;
+  min-width: 132px;
```

Comment:
This can be optional as there might be cases where there isn't enough space for respecting the `min-width`.